README.md

```diff
@@ -0,0 +1,16 @@
+# What is Speedometer?
+
+Speedometer is a benchmark for web browsers that measures Web application responsiveness
+by timing simulated user interactions on various workloads. Our primary goal is to make it
+reflect the real-world Web as much as possible. When a browser improves its score on the
```

Comment:
real-world Web _usage_?

Reply:
Yeah, it could be either "the real-world Web" or "real-world Web usage". This phrase is also reused at the bottom so should be consistent either way. I personally prefer the former but open to discussion about changing it (in a follow up, since the rest of the text has been thumbs upped)